### PR TITLE
Fix: Remediation text for the MCAST/UCAST check 1.1.6

### PR DESCRIPTION
**Pull request by aleksei-burlakov**
It improves the remediation text for the MCAST/UCAST check 1.1.6.

### CODE REVIEW

**Comment by aleksei-burlakov** [comment]
@lee-martin could you please have a look?

File: examples/generic-azure.yaml [hunk 1]
```diff
@@ -164,12 +164,37 @@ groups:
             - flag: "udpu"
         remediation: |
           ## Remediation
-          It will be difficult to change the corosync MCAST transport to UCAST
-          on an existing cluster. Besides adding the 'transport: udpu' parameter one
-          would have to configure each node separatelly. The easiest way to
-          make the cluster UCAST it to bootstrap it. When creating the cluster
-          you would have to add the key -u, for example
-          $ crm cluster init -u ...
+          To change the corosync MCAST transport to UCAST edit the /etc/corosync/corosync.conf
```
Comment (lee-martin) [inline examples/generic-azure.yaml:167]: So test id 1.1.6 checks if "Corosync transport parameter is set to udpu" , so how does that relate to documenting how to fix the MCAST/UCAST setting? It seems there are actually two separate tests at hand here:
1) Is udpu set correctly, and if not, how to remediate
2) Is UCAST set, and if not how to remediate.   The MCAST/UCAST text is improved, but that remediation belongs in the UCAST check (which I can't find in the xaml file, maybe needs creating?

File: examples/generic-azure.yaml [hunk 2]
```diff
@@ -164,12 +164,37 @@ groups:
...
+          +       }
+          +
+          +       node {
+          +               ring0_addr: 10.162.32.89
+          +               nodeid: 2
+          +       }
+          +
+          +}
+          ```
+          1) In the totem section, in the interface subsection remove the
```
Comment (lee-martin) [inline examples/generic-azure.yaml:193]: Reading the HANA HA setup guide, it seems the first step is  "stop the already running cluster by using systemctl stop pacemaker" ? See section 9.1.3 of https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/

File: examples/generic-azure.yaml [hunk 3]
```diff
@@ -164,12 +164,37 @@ groups:
...
+          keys-value pairs **bindnetaddr** and **mcastaddr**.
+          2) In the totem section add key-value pair **transport: udpu**
+          3) Add section nodelist and subsections node for each nodes of the
```
Comment (lee-martin) [inline examples/generic-azure.yaml:196]: Does the nodelist section apply to udpu, or only UCAST remediation?
As mentioned above, it seems two separate checks (udpu and UCAST)  got merged here.

File: examples/generic-azure.yaml [hunk 4]
```diff
@@ -164,12 +164,37 @@ groups:
...
+          cluster, where the **ring0_addr** is the IP address of the node.
 
           ## References
           - https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse-pacemaker
```
Comment (lee-martin) [inline examples/generic-azure.yaml:200]: Please always also reference SUSE guides where they exist.
In this case section 9.1.3 of https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/

Moreover I do not even find the UCAST recommendation at: https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse-pacemaker
Actually that MSFT guide does not even seem to mention udpu explicitly either. Can you have a second look to see if my eyes are missing these details, or if you looks a a different MSFT link? We really should avoid making incorrect claims about what sources recommend, unless I missed something or it was changed of course.

**Comment by lee-martin** [comment]
> @lee-martin could you please have a look?

@aleksei-burlakov  Sure, please see my comments above. Please comment me directly when you are done since I don't yet have a mail filter which sees when I'm being requested to review. This is still new to me :)

**Comment by lee-martin** [comment]
@aleksei-burlakov  thanks, this looks pretty good now. The only missing thing I can see is that the reference is " section 9.1.3 " specifically and not the general " section 9.1 ", but other than that, I'm good.

@arbulu89 & @stefanotorresi  following our conversation, it seems all we are missing here is the minor correction in the reference mentioned above, once we have that we are good to merge from my side

**Review by lee-martin — APPROVED** [review]
looks fine from my side now. thanks.

**Comment by aleksei-burlakov** [comment]
It fails, I think because the benchcommon checkers were removed.

**Comment by arbulu89** [comment]
> It fails, I think because the benchcommon checkers were removed.

`benchcommon` has only be removed in the `runner` branch. This `main` branch should unaffected

**Comment by arbulu89** [comment]
@lee-martin changes done for the ansible implementation too: https://github.com/trento-project/trento/pull/225

**Comment by arbulu89** [comment]
@aleksei-burlakov @lee-martin 
Closing this, as the text has been already ported to the ansible optoin

**Pull request closed without merging**